src/styles/components/cards.module.css

```diff
@@ -0,0 +1,17 @@
+.card {
+  backdrop-filter: blur(2rem);
+  -webkit-backdrop-filter: blur(2rem);
+}
+
+.card__white {
+  background: rgba(255, 255, 255, 0.3) !important;
+  border: 1px solid rgba(0, 0, 0, 0.1) !important;
+}
+
+.card__white:hover {
+  border: 1px solid rgba(0, 0, 0, 0.3) !important;
+}
+
+.card__dark {
+  background: rgba(0, 0, 0, 0.3) !important;
+}
```

Comment:
Well, dark mode already have a really nice border, so there's no need to change it. As for the light mode, yes it does need some enhancement.